### ISSUE TITLE
Crash: IDA 7.5 SP3 on macOS upon attempted decompilation with blc

### ISSUE THREAD

**Issue by kciredor** [cseagle/blc#8]
Hi @cseagle,

First of all: thanks for your efforts and releasing blc.

Just downloaded the SP3 release of IDA Pro 7.5, running on macOS Catalina 10.15.7 having GHIDRA_DIR=/usr/local/Caskroom/ghidra/9.1.2_PUBLIC,20200212/ghidra_9.1.2_PUBLIC.

Opened linux binary afl-fuzz to ensure blc still works. Unfortunately it does not. Everything was still fine with SP2. I've tried again using newly built binaries to no avail.

When you start decompilation with blc, IDA immediately shows you it's about te crash (Let it crash vs Continue).

Best,
kciredor

**Comment by kciredor** [comment]
It does not crash.with a random ARM64 binary I've just tried.

**Comment by alexander-pick** [comment]
Have you compiled the module for SP3 yourself, I don't get this part and which function did you try to decompile?

**Comment by kciredor** [comment]
Yes I also compiled it myself, this one and your fork as well @alexander-pick ;-)

Opened afl-fuzz MachO 64bit in IDA64 and did not change any defaults. Then on the public main function I press my hotkey to exec blc and it crashes.

**Comment by alexander-pick** [comment]
Is it just in the main or any other functions too?

**Comment by kciredor** [comment]
Any, for example just tried _setsid which is only a couple of lines of assembly.
